explorer/explorer_pods.go

```diff
@@ -40,6 +69,22 @@ func (n *PodsExplorer) List() error {
 	return nil
 }
 
+// gets pods from map of owners
+func getOwnerPods(pods []apiv1.Pod, owners map[string]bool, resourceKind string) []apiv1.Pod {
+	var podlist []apiv1.Pod
+
+	for _, pod := range pods {
+		for _, owner := range pod.GetOwnerReferences() {
+			if _, ok := owners[owner.Name]; ok && strings.ToLower(owner.Kind) == resourceKind {
+				podlist = append(podlist, pod)
+				break
```

Comment:
Why do we break here? I think we should continue and list all the pods

Reply:
Each pod has a list of owners. So we break as soon as we find an owner that matches in the pod's list of owners.